### DISCUSSION TITLE
[Question]: How zones work on PTZ cameras?

### DISCUSSION THREAD

**Discussion by matiasba** [blakeblackshear/frigate#13257] [Ask A Question] [answered]
### What is your question:

Are zones somehow linked to a preset or calibrated during startup on ptz cameras? I had to disable auto tracking because it deformed my zones after every startup.
If zones are static, how does the auto tracking handles camera movement and motion masks?

**Comment by hawkeye217** [comment, marked as answer]
Zones on PTZ cameras are usually not applicable - precisely because zones assume a fixed position. Autotracking uses zones only as a trigger. The PTZ starts is in its "home" position and frigate assumes it's a static camera until it begins moving. Autotracking doesn't use zones once it's actively tracking an object.

On my PTZ with autotracking, I have one zone (my lawn) and one motion mask (my camera's timestamp).

**Reply by matiasba** [reply]
If zones are not applied after auto tracking begins, how does the camera handles detection between target lost and return to home? It tries to search for the same type of object on the whole screen ?

**Reply by hawkeye217** [reply]
It looks for the same type of object in the [region](https://docs.frigate.video/frigate/glossary#region) where the object was lost.

**Reply by automation-dude-hass** [reply]
I know that for PTZ - full sensor zones are not recommended. Can you share a picture of how you have your zone configured for the PTZ @hawkeye217 ?

**Reply by hawkeye217** [reply]
My zone is nothing special - I don't want to autotrack people walking across the street at the neighbors, but I do want to autotrack people on the side of the road and lawn. So I just created a zone that covers all of that area.
